lib/crypto.js

```diff
@@ -199,6 +200,7 @@ module.exports = {
   randomFill,
   randomFillSync,
   randomInt,
+  randomUUID,
```

Comment:
Regarding naming: The function name `randomUUID()` was initially proposed and discussed in https://github.com/tc39/proposal-uuid/issues/3#issuecomment-543452495

We were also discussing renaming the api of the `uuid` npm module (https://github.com/uuidjs/uuid/issues/540#issuecomment-737758829) but came to the conclusion that we don't want to do that while the standardization efforts are still ongoing.

After all I still think the term `randomUUID` has a lot of benefits since it clearly describes what this is about whereas the whole concept of "version" within the UUID RFC has proven to be highly confusing, see [my detailed analysis](https://github.com/tc39/proposal-uuid/tree/master/analysis#analysis-of-uuid-usage-in-github-bigquery-dataset).

It also avoids naming conflicts with the current `uuid` npm module so this should remove confusion when upgrading from the npm module to the native implementation.

So all in all I'm 👍 for naming it like that in Node.js